Formula/d/decker.rb

```diff
@@ -0,0 +1,26 @@
+class Decker < Formula
+  desc "HyperCard-like multimedia sketchpad"
+  homepage "https://beyondloom.com/decker/"
+  url "https://github.com/JohnEarnest/Decker/archive/refs/tags/v1.56.tar.gz"
+  sha256 "395cf6968d23b9a36323597713743aac93b3e0aa8251359c403b747cfee893bd"
+  license "MIT"
+
+  depends_on "sdl2"
+  depends_on "sdl2_image"
+
+  on_linux do
+    depends_on "vim" => :build # uses xxd
+  end
+
+  def install
+    extra_flags = "-I#{HOMEBREW_PREFIX}/include/SDL2"
```

Comment:
Would this work?
```suggestion
    extra_flags = "-I#{Formula["sdl2"].opt_include}/include/SDL2"
```

Reply:
Unfortunately, that would require another `-I` flag for the `sdl2_image` include.